src/libraries/pkg/test/project.csproj.template

```diff
@@ -5,6 +5,8 @@
     <RuntimeIdentifiers>{RIDs}</RuntimeIdentifiers>
     <TestPackageId>{PackageId}</TestPackageId>
     <TestPackageVersion>{PackageVersion}</TestPackageVersion>
+    <!-- Turn off end of life target framework checks as we intentionally build older .NETCoreApp configurations. -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
We usually put these properties here: https://github.com/dotnet/runtime/blob/322b8f716a1795d273864febb56f37fd26b2c87e/src/libraries/pkg/test/props/Directory.Build.props#L4

Reply:
I tried that (see commits) but it didn't work.

Reply:
Hmm, ok, interesting. That seems like probably a bug on the pkg testing infrastructure; not critical for this change though.

Reply:
It still doesn't work in CI. Help is appreciated.

Reply:
Ok seems it worked when setting it in the project template.